sec/review.tex

```diff
@@ -438,16 +438,26 @@ \subsection{Healthcare modelling}\label{subsec:healthcare}
 populations~\cite{Foster2019,Gambling2010,Gondek2016,Tsianakas2012} and more
 broadly~\cite{IAPO2012,Richards2015,Santana2019}.
 
+The remainder of this section considers two aspects of healthcare modelling
+research that directly contribute to the advancement of patient-centred care:
```

Comment:
```suggestion
research that directly contributes to the advancement of patient-centred care:
```

Reply:
I think `contribute` is correct here: "two aspects [...] that *contribute* to..."